### PR TITLE
Update kitti dataset checksum files

### PR DESCRIPTION
**Pull request by vijayphoenix**
* Dataset Name: kitti dataset
* Issue Reference: https://github.com/tensorflow/datasets/issues/1221
* `dataset_info.json` Gist: [link](https://gist.github.com/vijayphoenix/cb778fc85a6f1fee3c1cf703ddd353c9)
  
  
## Checklist
* [X] Run `download_and_prepare` successfully
* [X] Add checksums file

CC: @Conchylicultor @cyfra, please review

### CODE REVIEW

**Comment by Conchylicultor** [comment]
Thank you very much for updating this. Could you just investigate a little why has the file changed ? (ex: has a new version of kitti been released ?).
In any case, we need to update the dataset Kitti version as this might have changed the underlying data.

It would be nice to link the change to this PR. No worries if you don't find anything.

**Comment by vijayphoenix** [comment]
So, I found this on their site.

> Note 2: On 08.10.2019, we have followed the suggestions of the Mapillary team in their paper Disentangling Monocular 3D Object Detection and use 40 recall positions instead of the 11 recall positions proposed in the original Pascal VOC benchmark. 

This might have caused the changes.

**Comment by vijayphoenix** [comment]
@Conchylicultor Updated the version to `3.1.1`

**Comment by Conchylicultor** [comment]
Thanks for the quick update.
In the gist you copied, only the validation set has been generated (train and test are missing), this means that the new file has probably changed its structure.

The dataset and fake files also need to be updated to reflect those new changes.

**Comment by vijayphoenix** [comment]
No, my computer ran out of space so I generated gist only for validation for now :-)

**Comment by vijayphoenix** [comment]
The `dataset_info.json` now contains both validation and test set.
Unable to provide info for training set because of storage limitations on my side.

**Comment by Conchylicultor** [comment]
Thank you for the update. I've merged this internally and it should appear in TFDS nightly in a few days
